packages/Manager/src/test/kotlin/nz/govt/eop/consumers/WaterAllocationConsumerTest.kt

```diff
@@ -8,30 +8,34 @@ import nz.govt.eop.si.jooq.tables.WaterAllocations.Companion.WATER_ALLOCATIONS
 import nz.govt.eop.si.jooq.tables.records.WaterAllocationsRecord
 import org.jooq.DSLContext
 import org.jooq.Result
+import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.kafka.core.KafkaTemplate
-import org.springframework.kafka.test.context.EmbeddedKafka
 import org.springframework.test.context.ActiveProfiles
+import org.springframework.transaction.annotation.Transactional
 
+// This test doesn't actually disable the Kafka listener, but it just doesn't use it.
 @ActiveProfiles("test")
 @SpringBootTest
-@EmbeddedKafka(
-    partitions = 1,
-    bootstrapServersProperty = "spring.kafka.bootstrap-servers",
-    topics = [WATER_ALLOCATION_TOPIC_NAME])
+@Transactional
```

Comment:
`@Transactional` makes it roll back any transaction at the end of the test ... meaning the DB shouldn't be changed

Reply:
Ah yeah ... it won't work for that one, which can make things a bit confusing.

The `@Transactional` works in the context that the test is in, theres kinda a transaction started for each test and the database things we use (e.g. DSLContext) get hooked into that transaction, because its all the same thread.

But In the integration test there are two transactions happening, one in the test and one for the consumer. So the one in the consumer wouldn't get the same magic